src/db/DB_Table_Currencyformats.h

```diff
@@ -109,285 +109,285 @@ struct DB_Table_CURRENCYFORMATS : public DB_Table
-        db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS VALUES ('270', '%s', '', '', '.', ',', '100', '1', 'ZAL', 'Fiat', '1')", wxTRANSLATE("Financial Rand (before 1995-03)")));
-        db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS VALUES ('271', '%s', '', '', '.', ',', '100', '1', 'ZMK', 'Fiat', '1')", wxTRANSLATE("Zambian Kwacha (before 2012-12)")));
-        db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS VALUES ('272', '%s', '', '', '.', ',', '100', '1', 'ZMW', 'Fiat', '0')", wxTRANSLATE("Zambian Kwacha")));
-        db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS VALUES ('273', '%s', '', '', '.', ',', '100', '1', 'ZRN', 'Fiat', '1')", wxTRANSLATE("New Zaire (before 1999-06)")));
-        db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS VALUES ('274', '%s', '', '', '.', ',', '100', '1', 'ZRZ', 'Fiat', '1')", wxTRANSLATE("Zaire (before 1994-02)")));
-        db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS VALUES ('275', '%s', '', '', '.', ',', '100', '1', 'ZWC', 'Fiat', '1')", wxTRANSLATE("Rhodesian Dollar (before 1989-12)")));
-        db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS VALUES ('276', '%s', '', '', '.', ',', '100', '1', 'ZWD', 'Fiat', '1')", wxTRANSLATE("Zimbabwe Dollar (before 2008-08)")));
-        db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS VALUES ('277', '%s', '', '', '.', ',', '100', '1', 'ZWL', 'Fiat', '0')", wxTRANSLATE("Zimbabwe Dollar")));
-        db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS VALUES ('278', '%s', '', '', '.', ',', '100', '1', 'ZWN', 'Fiat', '1')", wxTRANSLATE("Zimbabwe Dollar (new) (before 2006-09)")));
-        db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS VALUES ('279', '%s', '', '', '.', ',', '100', '1', 'ZWR', 'Fiat', '1')", wxTRANSLATE("Zimbabwe Dollar (before 2009-06)")));
+        db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS VALUES ('1', '%s', '$', '', '.', ' ', '100', 'USD', 'Fiat', '0')", wxTRANSLATE("US Dollar")));
+        db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS VALUES ('2', '%s', '%s', '', '.', ' ', '100', 'EUR', 'Fiat', '0')", wxTRANSLATE("Euro"), L"€"));
+        db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS VALUES ('3', '%s', '%s', '', '.', ' ', '100', 'GBP', 'Fiat', '0')", wxTRANSLATE("Pound Sterling"), L"£"));
+        db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS VALUES ('4', '%s', '', '%s', ',', ' ', '100', 'RUB', 'Fiat', '0')", wxTRANSLATE("Russian Ruble"), L"р"));
+        db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS VALUES ('5', '%s', '%s', '', ',', ' ', '100', 'UAH', 'Fiat', '0')", wxTRANSLATE("Hryvnia"), L"₴"));
+        db->ExecuteUpdate(wxString::Format("INSERT INTO CURRENCYFORMATS VALUES ('6', '%s', '%s', '', '.', ' ', '100', 'AFN', 'Fiat', '0')", wxTRANSLATE("Afghani"), L"؋"));
```

Comment:
here is ok

Reply:
Maybe is something related to repository autoencoding?